app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,10 @@
+<footer class="footer">
+
+  <nav>
+    <ul>
+      <li><%= link_to t('footer.about_link'), '#' %></li>
```

Comment:
Coding convention: file html, rb dùng nháy đôi trc này, self check file này và toàn bộ các file html.erb bên dưới